src/twisted/newsfragments/12325.bugfix

```diff
@@ -0,0 +1 @@
+TLS endpoint strings with "bindAddress=" should no longer crash during connect. The HostnameEndpoint() "bindAddress" argument now accepts either hostnames or (host,port) tuples.
```

Comment:
I a coding standard for the release note, we try to mention the public API affected by this change.

I guess that this is `clientFromString`, but I am not sure.

Also, we have tests, so we are sure it works :) I don't think that we need the "should" part 


```suggestion
twisted.internet.endpoints.clientFromString for TLS endpoint with "bindAddress=" no longer crashes during connect. The HostnameEndpoint() "bindAddress" argument now accepts either hostnames or (host,port) tuples.
```

Reply:
how's this:
```
twisted.internet.endpoints.clientFromString for TLS endpoints with "bindAddress="  no longer crashes during 
connect. twisted.internet.endpoints.HostnameEndpoint() no longer crashes when given a bindAddress= 
argument that is just a string, and that argument now accepts either address strings or (address, port) tuples.
```